packages/@okta/vuepress-site/docs/reference/api/risk-providers/index.md

```diff
@@ -0,0 +1,209 @@
+---
+title: RiskProviders
+category: management
+---
+
+# Risk Providers API
+
+This API providers operations to manage the risk providers within Okta.
+
+## Risk Provider object
```

Comment:
This should go at the bottom of the page. Also, since this is an object within Okta, all instances of it should be capitalized, eg: `to manage Risk Providers within Okta`